lib/publiccloud/basetest.pm

```diff
@@ -33,22 +33,22 @@ sub provider_factory {
 
         if ($args{service} eq 'ECR') {
             $provider = publiccloud::ecr->new(
-                key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
-                key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
+                # key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
+                # key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
                 region => get_var('PUBLIC_CLOUD_REGION', 'eu-central-1')
             );
         }
         elsif ($args{service} eq 'EKS') {
             $provider = publiccloud::eks->new(
-                key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
-                key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
+                # key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
+                # key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
                 region => get_var('PUBLIC_CLOUD_REGION', 'eu-central-1')
```

Comment:
Why isn't this moved to `lib/publiccloud/eks.pm`?

Reply:
Same here